src/main/java/com/griefcraft/model/Protection.java

```diff
@@ -32,28 +32,18 @@
 import com.griefcraft.cache.ProtectionCache;
 import com.griefcraft.lwc.LWC;
 import com.griefcraft.scripting.event.LWCProtectionRemovePostEvent;
-import com.griefcraft.util.Colors;
-import com.griefcraft.util.ProtectionFinder;
-import com.griefcraft.util.StringUtil;
-import com.griefcraft.util.TimeUtil;
-import com.griefcraft.util.UUIDRegistry;
+import com.griefcraft.util.*;
```

Comment:
Please don't use wildcard imports. You can change the setting in your IDE to disable this if it is caused by auto format.